### PR TITLE
Add `Yaz's Summer Cup 2017` tournament article and other prerequisites 

### PR DESCRIPTION
**Pull request by Bazzadwarf**
The first out of four PR's for adding Yaz's Summer Cup to the wiki, it's a large one so be aware.

TODO's
- [x] Fix's formatting inconsistencies 
- [x] Check by yaz

### CODE REVIEW

File: wiki/Tournaments/YSC/2017/en.md [hunk 1]
```diff
@@ -0,0 +1,924 @@
...
+- Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.
+- Each stage will be held on a friday/saturday/sunday. Schedules and mappools will be released on the monday/tuesday before that stage is due to happen, to allow for reschedules.
+- reschedules will be allowed if both players agree on a time and provide proof of the agreed reschedule to hosts/co-hosts/scheduling team.
+- Please be aware that reschedules must happen at least 24 hours before the match is due to happen. Any time after this will not be considered.
+- In order for your registration to count, you must fill out the form provided, join the discord and change your username on discord to your osu! Username.
+
+### General Rules
+
+- Referees will create the lobby and invite players 10 minutes before the match is due to start.
+- If a player fails to show up after their match is due to start, a 10 minute timer will be started.
```
Comment (Walavouchey) [inline wiki/Tournaments/YSC/2017/en.md:853]: redundant (combined with below bullet point)

```suggestion
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 2]
```diff
@@ -0,0 +1,924 @@
...
+- Any player that fails to show up after the 10 minute timer will forfeit the match and a default win will be awarded to the other player.
```
Comment (Walavouchey) [inline wiki/Tournaments/YSC/2017/en.md:854]:
```suggestion
- Any player that fails to show up 10 minutes after a match is due to start will forfeit the match and a win by default will be awarded to the other player.
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 3]
```diff
@@ -0,0 +1,924 @@
...
+- The score of a player who is in failed state when the map has finished will be counted as 0.
+- If both players fail, the highest failed score will win the map.
+- If there is a tie, the tiebreaker will be played to determine the winner of that match (mods are available during the tiebreaker).
+- At the end of the match, you are free to leave the lobby.
+
+### Qualifiers and Group Stage
+
+- There will be no warmups and one ban per player.
+- Players will roll once. The winner of that roll will ban and pick first.
+- In groups, there will be one top, one high, one mid and one low seeded player per group, this will be drawn randomly.
```
Comment (Walavouchey) [inline wiki/Tournaments/YSC/2017/en.md:873]:
```suggestion
- In groups, there will be one top, one high, one mid and one low seeded player per group, which will be drawn randomly.
```

**Review by Walavouchey — COMMENTED** [review]
writing is alright otherwise

File: wiki/Tournaments/YSC/2017/en.md [hunk 4]
```diff
@@ -0,0 +1,924 @@
...
+| --: | :-: | :-: | :-- | :-- |
+| Rohulk ![][flag_RO] | 4 | **6** | ![][flag_US] **Vaxei** |  |
+
+Sunday, 3 September 2017:
+
+| Player 1 | | | Player 2 | Match info |
+| --: | :-: | :-: | :-- | :-- |
+| **Toy** ![][flag_US] | **6** | 2 | ![][flag_US] he rack |  |
+
+#### Losers Bracker Round 4
```
Comment (MilkyIQ) [inline wiki/Tournaments/YSC/2017/en.md:382]: do these intend to say "bracket"

File: wiki/Tournaments/YSC/2017/en.md [hunk 5]
```diff
@@ -0,0 +1,924 @@
...
+## Ruleset
+
+- This tournament is a osu!, 1v1, international tournament.
+- This tournament will consist of 128 players.
+- Matches will be head to head, with score v2 as the scoring system used.
+- There will be no rank limit on signups, however if we reach 128 signups we will take the top 128 players in terms of rank.
+- Ranks will be taken and locked when registrations have concluded
+- Anyone who does sign up must be able to play maps with a difficulty of at least 4.7*.
+- Starting from groups, the tournament will be double elimination. Qualifications will be the only thing that is single elimination.
+- All staff members are allowed to sign up for the tournament. The only exception are the host, co-hosts, and mappool selectors/testers.
```
Comment (MilkyIQ) [inline wiki/Tournaments/YSC/2017/en.md:842]:
```suggestion
- All staff members are allowed to sign up for the tournament. The only exceptions are the host, co-hosts, and mappool selectors/testers.
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 6]
```diff
@@ -0,0 +1,924 @@
...
+- All players must treat staff and other participants with a high level of respect, any kind of mistreatment in a match or on the discord (this includes extreme toxicity, racism, homophobia) will result in ban and will be blacklisted from participating in future installments of this tournament.
```
[the remark this answers is not in the document]
Comment (MilkyIQ) [reply wiki/Tournaments/YSC/2017/en.md:843]:
```suggestion
- All players must treat staff and other participants with a high level of respect, any kind of mistreatment in a match or on the Discord server (this includes extreme toxicity, racism, homophobia) will result in a ban and will be blacklisted from participating in future installments of this tournament.
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 7]
```diff
@@ -0,0 +1,924 @@
...
+- Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.
+- Each stage will be held on a friday/saturday/sunday. Schedules and mappools will be released on the monday/tuesday before that stage is due to happen, to allow for reschedules.
```
[the remark this answers is not in the document]
Comment (MilkyIQ) [reply wiki/Tournaments/YSC/2017/en.md:845]:
```suggestion
- Each stage will be held on a Friday/Saturday/Sunday. Schedules and mappools will be released on the Monday/Tuesday before that stage is due to happen to allow for reschedules.
```
also consider removing pause-comma; nitpick

File: wiki/Tournaments/YSC/2017/en.md (hunk 4 above)
Comment (Bazzadwarf) [reply wiki/Tournaments/YSC/2017/en.md]: Yes

**Review by MilkyIQ — COMMENTED** [review]
a few grammar suggestions

**Comment by Bazzadwarf** [comment]
Got the ok from yaz, ready for review!

File: wiki/Tournaments/YSC/2021/en.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Yaz's Summer Cup 2021
+
+The **Yaz's Summer Cup 2021** (***YSC 2021***) is a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It is the 4th installment of Yaz's Summer Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2021/en.md:3]:
```suggestion
The **Yaz's Summer Cup 2021** (***YSC 2021***) is a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It is the fourth instalment of Yaz's Summer Cup.
```

File: wiki/Tournaments/YSC/2019/en.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Yaz's Summer Cup 2019
+
+The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the 3rd installment of Yaz's Summer Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2019/en.md:3]:
```suggestion
The **Yaz's Summer Cup 2019** (***YSC 2019***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the third instalment of Yaz's Summer Cup.
```

File: wiki/Tournaments/YSC/2018/en.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# Yaz's Summer Cup 2018
+
+The **Yaz's Summer Cup 2018** (***YSC 2018***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the 2nd installment of Yaz's Summer Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2018/en.md:3]:
```suggestion
The **Yaz's Summer Cup 2018** (***YSC 2018***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the second instalment of Yaz's Summer Cup.
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 11]
```diff
@@ -0,0 +1,924 @@
+# Yaz's Summer Cup 2017
+
+![YSC 2017 logo](img/logo.png)
+
+The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the 1st installment of Yaz's Summer Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:5]:
```suggestion
The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the first instalment of Yaz's Summer Cup.
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 12]
```diff
@@ -0,0 +1,924 @@
...
+| Grand Finals | 2017-09-30 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $60, profile badge, 6 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $35, 4 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $20, 2 months of osu!supporter tag |
+| *4th place* | $10, 1 month of osu!supporter tag |
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:29]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $60, profile badge, 6 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $35, 4 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $20, 2 months of osu!supporter |
| *4th place* | $10, 1 month of osu!supporter |
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 13]
```diff
@@ -0,0 +1,924 @@
...
+
+![](img/badge.png "YSC 2017 winner badge")
+
+## Organisation
+
+Yaz's Summer Cup 2017 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:35]:
```suggestion
Yaz's Summer Cup 2017 was run by various community members.
```
can use this version, we deprecated the long one pretty much

File: wiki/Tournaments/YSC/2017/en.md [hunk 14]
```diff
@@ -0,0 +1,924 @@
...
+| Group O | ![][flag_US] **[Monko2k](https://osu.ppy.sh/users/4852013)**, ![][flag_US] **[Smoothie World](https://osu.ppy.sh/users/1855540)**, ![][flag_NL] [Timpai](https://osu.ppy.sh/users/5054798), ![][flag_RU] [ncuh](https://osu.ppy.sh/users/2891734) |
+| Group P | ![][flag_ES] **[Betwin](https://osu.ppy.sh/users/1968481)**, ![][flag_GB] **[Helix](https://osu.ppy.sh/users/3322597)**, ![][flag_GB] [TopSp1n](https://osu.ppy.sh/users/7657761), ![][flag_EG] [KupcaH](https://osu.ppy.sh/users/7844575) |
+
+See the [participants sheet](https://docs.google.com/spreadsheets/d/1cO_DR2ffyp0HuGtlHUHEV0MqQR-I0l7AgkQdygjjNmw/pubhtml?gridlines=false "Google Sheets") for a list of all 128 initial participants.
+
+## Mappools
+
+### Grand Finals
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:85]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 15]
```diff
@@ -0,0 +1,924 @@
...
+  - [Ling tosite sigure - KIMITOOKU (Milan-) [pishi's Extra]](https://osu.ppy.sh/beatmapsets/458931#osu/988684)
+  - [100-200-400 - Fascination MAXX (Mystearica) [28 KEY]](https://osu.ppy.sh/beatmapsets/15650#osu/56549)
+  - [C-SHOW - Workaholic (ktgster) [Extra]](https://osu.ppy.sh/beatmapsets/103470#osu/272645)
+  - [Falcom Sound Team jdk - RELEASE OF THE FAR WEST OCEAN (CSY the corrupt) [Corruption]](https://osu.ppy.sh/beatmapsets/77504#osu/217493)
+- Tiebreaker
+  - **[BlackYooh vs. siromaru - BLACK or WHITE? (Akali) [GRANDMASTER]](https://osu.ppy.sh/beatmapsets/463454#osu/992389)**
+
+### Finals
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:117]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 16]
```diff
@@ -0,0 +1,924 @@
...
+  - [sakuzyo - Imprinting (EvilElvis) [Extreme]](https://osu.ppy.sh/beatmapsets/370843#osu/812665)
+  - [Rita - Beyond the Eternity (Delis) [Desolation]](https://osu.ppy.sh/beatmapsets/630897#osu/1331749)
+  - [Sakuzyo - Laplace (yf\_bmp) [AR8]](https://osu.ppy.sh/beatmapsets/427582#osu/923912)
+  - [Tonight Alive - Little Lion Man (puxtu) [pishi's Extra]](https://osu.ppy.sh/beatmapsets/451311#osu/1046775)
+- Tiebreaker
+  - **[RIN - Pluto -Tenderness (My Angel Azusa) [Hades In The Heaven]](https://osu.ppy.sh/beatmapsets/573313#osu/1241443)**
+
+### Semifinals
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:149]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 17]
```diff
@@ -0,0 +1,924 @@
...
+  - [ginkiha+kamome sano - Dawn of Asia (Rare) [INFINITE]](https://osu.ppy.sh/beatmapsets/122041#osu/596081)
+  - [BABYMETAL - MEGITSUNE (Onisakura) [BluOxy's Challenging]](https://osu.ppy.sh/beatmapsets/110357#osu/343528)
+  - [Sharlo - Hatsukoi no Ehon (Minakami Yuki) [Extra]](https://osu.ppy.sh/beatmapsets/332887#osu/736886)
+  - [HujuniseikouyuuP - Talent Shredder (val0108) [0108 style]](https://osu.ppy.sh/beatmapsets/47710#osu/178966)
+- Tiebreaker
+  - **[Meramipop - Eiya [Imperishable Challengers] (-Tochi) [Eternal Night]](https://osu.ppy.sh/beatmapsets/289225#osu/652034)**
+
+### Quarterfinals
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:181]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 18]
```diff
@@ -0,0 +1,924 @@
...
+  - [Fall Out Boy - Alone Together (Krewella Remix) (Aia) [Extra]](https://osu.ppy.sh/beatmapsets/290995#osu/784411)
+  - [Dreamcatcher - GOOD NIGHT (Natsu) [toybot's Sunset]](https://osu.ppy.sh/beatmapsets/603905#osu/1286416)
+  - [jippusu - Mushikui Saikede Rhythm (Amamiya Yuko) [Skystar]](https://osu.ppy.sh/beatmapsets/87547#osu/239104)
+  - [Giga-P - LUVORATORRRRRY! (Selisi) [HW's Extra]](https://osu.ppy.sh/beatmapsets/153206#osu/394808)
+- Tiebreaker
+  - **[Bullet For My Valentine - Tears Don't Fall (FCL) [The Poison]](https://osu.ppy.sh/beatmapsets/374922#osu/821273)**
+
+### Round of 16
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:213]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 19]
```diff
@@ -0,0 +1,924 @@
...
+  - [bj.HaLo - Ende (galvenize) [Another]](https://osu.ppy.sh/beatmapsets/44035#osu/148716)
+  - [Itou Kanako - Skyclad no Kansokusha -Remix- (Takos) [taka]](https://osu.ppy.sh/beatmapsets/33323#osu/110628)
+  - [Aiobahn & Yunomi - Ginga Tetsudou no Penguin ft. nicamoq (Stripe.P Remix) (deetz) [Collab]](https://osu.ppy.sh/beatmapsets/442510#osu/951600)
+  - [Ryu\* - Mind Mapping (kors k mix) (wring) [EXTRA]](https://osu.ppy.sh/beatmapsets/47199#osu/146583)
+- Tiebreaker
+  - **[Akira Complex - Odyssey (Au5 Remix) (ProfessionalBox) [Excursion]](https://osu.ppy.sh/beatmapsets/341007#osu/754143)**
+
+### Round of 32
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:245]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 20]
```diff
@@ -0,0 +1,924 @@
...
+  - [Samantha Jade - Soldier (7th Heaven Remix) (Rakuen) [Flask's Insane]](https://osu.ppy.sh/beatmapsets/157234#osu/716691)
+  - [Nightwish - She Is My Sin (NatsumeRin) [Insane]](https://osu.ppy.sh/beatmapsets/13445#osu/49748)
+- FreeMod
+  - [Masayoshi Minoshima - Flight of the Bamboo Cutter (a3272509123) [Lunatic]](https://osu.ppy.sh/beatmapsets/47754#osu/148000)
+  - [Hikasa Yoko - Listen!! (CDFA) [Collab]](https://osu.ppy.sh/beatmapsets/24610#osu/83603)
+  - [NegaRen - Stark Raving Mad (vipto) [Raving Mad]](https://osu.ppy.sh/beatmapsets/54618#osu/166350)
+- Tiebreaker
+  - **[Dark PHOENiX - Green-Eyed Jealousy (sjoy) [Envy]](https://osu.ppy.sh/beatmapsets/170913#osu/413597)**
+
+### Group Stage
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:269]:
```suggestion
### Group stage
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 21]
```diff
@@ -0,0 +1,924 @@
...
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:271]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 22]
```diff
@@ -0,0 +1,924 @@
...
+- FreeMod
+  - [Krewella - Come & Get It (Nightcore Mix) (Asphyxia) [Insane]](https://osu.ppy.sh/beatmapsets/125553#osu/348605)
+  - [Traktion - Mission ASCII (galvenize) [Another]](https://osu.ppy.sh/beatmapsets/53249#osu/162405)
+  - [Seiryu - Ultramarine (RLC) [Another]](https://osu.ppy.sh/beatmapsets/107377#osu/281389)
+- Tiebreaker
+  - **[Shoji Meguro - Kimi no Kioku (Aethral Remix) (Akali) [Remembrance]](https://osu.ppy.sh/beatmapsets/490662#osu/1045757)**
+
+### Qualifiers / Round of 128
+
+- Nomod
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:297]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 23]
```diff
@@ -0,0 +1,924 @@
...
+  - [Toshimasa - Soshite Kyou mo Koumakan no Tobira wa Hiraku (xxheroxx) [Sallad's Lunatic]](https://osu.ppy.sh/beatmapsets/51558#osu/159851)
+  - [ClariS - STEP (Rizia) [Nisestep]](https://osu.ppy.sh/beatmapsets/312851#osu/707636)
+- FreeMod
+  - [Demetori - Hoshi no Utsuwa \~ Casket of Star (happy30) [sjoy's Insane]](https://osu.ppy.sh/beatmapsets/139677#osu/1115583)
+  - [MUZIK SERVANT feat. TAM - Upright Blue (Reiji-RJ) [Collab]](https://osu.ppy.sh/beatmapsets/56450#osu/170677)
+  - [Tia - The Glory Days (Nara) [P A N's Insane]](https://osu.ppy.sh/beatmapsets/391953#osu/903567)
+- Tiebreaker
+  - **[Dancing Dolls - monochrome(Asterisk Makina Remix) (monstrata) [Scythe]](https://osu.ppy.sh/beatmapsets/349445#osu/770576)**
+
+## Match Results
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:321]:
```suggestion
## Match results
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 24]
```diff
@@ -0,0 +1,924 @@
...
+- All players must treat staff and other participants with a high level of respect, any kind of mistreatment in a match or on the discord (this includes extreme toxicity, racism, homophobia) will result in a ban and will be blacklisted from participating in future installments of this tournament.
+- Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.
+- Each stage will be held on a Friday/Saturday/Sunday. Schedules and mappools will be released on the Monday/Tuesday before that stage is due to happen, to allow for reschedules.
+- Reschedules will be allowed if both players agree on a time and provide proof of the agreed reschedule to hosts/co-hosts/scheduling team.
+- Please be aware that reschedules must happen at least 24 hours before the match is due to happen. Any time after this will not be considered.
+- In order for your registration to count, you must fill out the form provided, join the Discord server and change your Discord username to your osu! username.
+
+### General Rules
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:851]:
```suggestion
### General rules
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 25]
```diff
@@ -0,0 +1,924 @@
...
+- Players are only allowed to disconnect once per match. Anything after this will result in the match being forfeited to the other player.
+- Any attempts to stall the match, show toxicity or spam during the match will result in a warning. Failure to comply with this rule again will result in a point being awarded to another player.
+- In a FreeMod pick, both players must choose a mod (HD/HR/EZ) or a combination of these mods.
+- The score of a player who failed during a map but managed to revive before the end will be counted as if it were a pass.
+- The score of a player who is in failed state when the map has finished will be counted as 0.
+- If both players fail, the highest failed score will win the map.
+- If there is a tie, the tiebreaker will be played to determine the winner of that match (mods are available during the tiebreaker).
+- At the end of the match, you are free to leave the lobby.
+
+### Qualifiers and Group Stage
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:869]:
```suggestion
### Qualifiers and Group stage
```

File: wiki/Tournaments/YSC/2017/en.md [hunk 26]
```diff
@@ -0,0 +1,924 @@
...
+
+- There will be no warmups and one ban per player.
+- Players will roll once. The winner of that roll will ban and pick first.
+- In groups, there will be one top, one high, one mid and one low seeded player per group, which will be drawn randomly.
+- During these stages, players are not allowed to choose from the same mod bracket (HD/HR/DT/FreeMod) twice in a row. This excludes the NoMod bracket pool.
+- The top 2 players from each group will make it through to the knockout stage.
+
+### Knockout Stages
```
Comment (TPGPL) [inline wiki/Tournaments/YSC/2017/en.md:877]:
```suggestion
### Knockout stages
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**